src/app/Models/QuizOption.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class QuizOption extends Model
+{
+    use HasFactory;
+    protected $table = 'quiz_question_options';
```

Comment:
Again, if I am misunderstanding or missing the point, let me know.

Similarly to the `Quiz` model, I'd apply the convention:
Model Name: Singular
Table of Model: Model Name Pluralized

I **suspect** using `php artisan make:model ModelName -a` should create the appropiate table name in the migration.